docs/source/queries/timeouts.md

```diff
@@ -0,0 +1,48 @@
+# Query timeouts
+
+Query execution time can be limited by setting a request timeout. If a query does not complete
+in the given time, then `QueryError::RequestTimeout` is returned by the driver immediately,
+so that application logic can continue operating, but the query may still be in progress on the server.
+
+As a side note, if one wishes custom server-side timeouts (i.e. actual interruption of query processing),
+one can use a[`USING TIMEOUT <duration>` directive supported in ScyllaDB](https://github.com/scylladb/scylladb/blob/master/docs/cql/cql-extensions.md#using-timeout)
+(but not in Cassandra).
+
+Timeout can be set globally (per session) or locally (for given statement).
+The default per-session timeout is currently 30s.
+It is possible to turn off timeouts completely by providing `None` as timeout when building `Session`.
+However, setting per-statement timeout to `None` results in falling back to per-session timeout.
+
+```rust
+# extern crate scylla;
+# use std::error::Error;
+# async fn timeouts() -> Result<(), Box<dyn Error>> {
+    use scylla::{Session, SessionBuilder, query::Query};
```

Comment:
Please also remove the indentation.
In the rendered book there is empty space on the left but no apparent reason for it.